lib/Form/Renderer.php

```diff
@@ -26,6 +26,12 @@ public function __construct(
     $this->blocksRenderer = $blocksRenderer;
   }
 
+  public function render(array $form = []): string {
```

Comment:
I am not sure if we need this method here. If it is used only in one place we can use those other methods as well in `lib/Form/DisplayFormInWPContent.php`. We could maybe in the future split this class further because `renderStyles` and `renderHtml` have little in common and could be separated.